addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -126,6 +127,29 @@ export class LinkPopover extends Component {
             showReplaceTitleBanner: this.props.showReplaceTitleBanner,
             showLabel: !linkElement.childElementCount,
             stripDomain: true,
+            showAdvanceOptions: false,
+            relAttributesOptions: {
+                nofollow: {
+                    label: "nofollow",
+                    description: _t("Tells search engines not to follow this link"),
+                    isChecked: currentRelValues.includes("nofollow"),
+                },
+                noreferrer: {
+                    label: "noreferrer",
+                    description: _t("Tells search engines not to follow this link"),
```

Comment:
These two descriptions are identical. How are users going to differentiate between them?